server/routes/profile.js

```diff
@@ -56,14 +68,15 @@ const createProfileRouter = ({ offenderService }) => {
         link: '/content/4204',
       },
       visitsSummary: {
-        error: visitsError,
+        error: visitsError || visitsRemainingError,
         hasNextVisit,
         nextVisit,
         visitType,
         visitorName,
         startTime,
         endTime,
         link: '/content/4203',
+        visitsRemaining: remainingPvo + remainingVo,
```

Comment:
meh.. could centralize this logic by potentially pushing it down to the service layer.  

`getVisitsRemaining` rather than `getVisitsBalance`

Reply:
I originally did and changed it to return all the values. I guess it boils down to where you want the business logic to lie. I figured this way, the decision about which data to use lies within _profile.js_.